### ISSUE TITLE
Notice in adminsearch controller

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#28796]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

There is  notice when you do a targeted search without results in the search bar.

```
Notice à la ligne 207 du fichier /var/www/html/800/PrestaShop/var/cache/dev/smarty/compile/01/2a/3a/012a3aacf3577da6fea68ba6f9e8fd189e40dca1_0.file.view.tpl.php
[8] Undefined index: searchPanels

Notice à la ligne 207 du fichier /var/www/html/800/PrestaShop/var/cache/dev/smarty/compile/01/2a/3a/012a3aacf3577da6fea68ba6f9e8fd189e40dca1_0.file.view.tpl.php
[8] Trying to get property 'value' of non-object
```

![Screenshot (38)](https://user-images.githubusercontent.com/5064590/174610473-95fbe5d7-6ae3-4caf-965a-0bf0d4bc8fd7.png)

### Expected behavior

No notice :) 

### Steps to reproduce

Go to back office 

In the search bar select invoice and enter XXXX

See the notice 

### PrestaShop version(s) where the bug happened

178x 8.0.0

### PHP version(s) where the bug happened

7.4

**Comment by AureRita** [comment]
Hi @okom3pom 

Thank you for the reporting Issue.
I reproduced it on 8.0.x and not on 1.7.7.8 as you can see on this video :

https://user-images.githubusercontent.com/100699445/174633030-611ebaf8-2ef2-402d-8622-1380da8fb19b.mp4


It's a regression ! I will add it to the Patch Backlog to be fixed 

Personnally, I didn't reproduce it on 1.7.8.6,
Thank you for the PR ! 

Thanks

**Comment by ghost** [comment]
Ok i need to change my branch ? i target develop ?

**Comment by AureRita** [comment]
I think it can be better to target 8.0.x

**Comment by Progi1984** [comment]
Fixed by @okom3pom in #28802 and tested by @sLorenzini
